### PR TITLE
[No Ticket] - Fix HelpDrawerToggle React Warning

### PR DESCRIPTION
**Pull request by erinrep**
### Fixed
There is a React warning happening for the HelpDrawerToggle:
```
Warning: React does not recognize the `helpDrawerOpen` prop on a DOM element. If you intentionally want it to appear in the DOM as a custom attribute, spell it as lowercase `helpdraweropen` instead. If you accidentally passed it from a parent component, remove it from the DOM element.
          in button (created by Button)
          in Button (created by HelpDrawerToggle)
          in HelpDrawerToggle (at HelpDrawerWrapper.tsx:69)
```
The `helpDrawerOpen` prop was being applied to the Button component which it shouldn't be, so I added it to line 17 to pull it out from the others variable before others is applied to the Button

### CODE REVIEW

**Review by nichia — APPROVED** [review]
LGTM. Thanks

**Comment by erinrep** [comment]
@nichia what's your process for merging after an approval? Should I wait until all reviewers have approved or go ahead and merge?

**Review by line47 — APPROVED** [review]

**Comment by nichia** [comment]
@erinrep We typically wait for 2 reviews. But for a small change, we do merge after one reviewer has approved. This PR looks ready for merging.

File: packages/design-system/src/components/HelpDrawer/HelpDrawerToggle.jsx
Comment (bernardwang) [inline packages/design-system/src/components/HelpDrawer/HelpDrawerToggle.jsx:17]: This is something we forgot to do in the past, but we should also update `HelpDrawerToggle.d.ts` to use the React.HTMLProps for the button element considering we allow props to be passed down.

Something like this: 
```
export default class HelpDrawerToggle extends React.Component<
React.HTMLProps<HTMLButtonElement> & HelpDrawerToggleProps,
  any
> {
  render(): JSX.Element;
}
```

Comment (bernardwang) [reply packages/design-system/src/components/HelpDrawer/HelpDrawerToggle.jsx:17]: @nichia or I can also handle this if you are busy @erinrep, just let us know!

Comment (erinrep) [reply packages/design-system/src/components/HelpDrawer/HelpDrawerToggle.jsx:17]: @bernardwang that would be great, I've got a lot on my plate today. Thanks!

**Comment by bernardwang** [comment]
@line47 @nichia This is ready to merge after testing the new typescript changes

**Review by line47 — APPROVED** [review]
Looks good! Thanks @bernardwang

**Review by nichia — APPROVED** [review]
Thanks for picking that up. LGTM

**Pull request merged**
